packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -151,8 +151,10 @@ const createHooksCallback = createMembraneMarshall();
 
 export default function createVirtualEnvironment(
     globalObjectShape: object,
+    globalObjectVirtualizationTarget: WindowProxy & typeof globalThis,
     providedOptions?: BrowserEnvironmentOptions
 ): VirtualEnvironment {
+    validateRequiredObjects(globalObjectShape, globalObjectVirtualizationTarget);
```

Comment:
☝️ Let's get Java level of naming descriptive here since it's for very specific kinds of objects or have 2 validate functions (1 for each type with descriptive name)

Reply:
Cool, will do